plugins/win-capture/game-capture.c

```diff
@@ -735,13 +739,22 @@ static void game_capture_update(void *data, obs_data_t *settings)
 		free_whitelist(&gc->auto_capture);
 	}
 	
-	const char *img_path = obs_data_get_string(settings, SETTING_PLACEHOLDER_IMG);
+	const char *img_path = NULL;
+	const char *placeholder_text = NULL;
+	bool customize_placeholder = obs_data_get_bool(settings, SETTING_PLACEHOLDER_USE);
```

Comment:
I would rename this variable to `use_custom_placeholder`

Reply:
done. 
now name use_custom_placeholder used in both functions what work with this param.

Reply:
nice, perfect